cli/pkg/utils/utils.go

```diff
@@ -225,4 +225,67 @@ func ConvertMapI2MapS(v interface{}) interface{} {
 	}
 
 	return v
-}
\ No newline at end of file
+}
+
+type dbDetails struct {
+	DbHost     string
+	DbUser     string
+	DbPort     string
+	DbPassword string
+	SslMode    string
+	DbName     string
+}
+
+func SplitDBURI(dbURI string) (dbDetails, error) {
```

Comment:
Instead of writing this from scratch can we simply use https://github.com/jackc/pgconn/blob/master/config.go#L191